pkg/objectstate/rte/rte.go

```diff
@@ -96,6 +96,19 @@ func (em *ExistingManifests) MachineConfigsState(mf rtemanifests.Manifests, inst
 	return ret
 }
 
+func getMachineConfigLabel(mcp *machineconfigv1.MachineConfigPool) map[string]string {
```

Comment:
actually reading the code, I think we should just fail and put the conditions of the object in Degraded state.
This may require some internal refactoring, but better to avoid guesswork if we can help it.

Reply:
The problem is that we totally can have machine config pools as one that I specified under the description and I am assuming that we will want to use PAO together with the NRO.
Do you have any ideas on how to handle the above machine config pool?

Reply:
Ah, right, that's another story entirely. We need to think about this